packages/ui/src/routes/persona/new/+page.svelte

```diff
@@ -36,13 +40,11 @@
 </script>
 
 {#if showWarningModal}
-	<InfoScreen title="Leaving persona creation" onBack={() => (showWarningModal = false)}>
-		<div>
-			<h1>Are you sure you want to leave?</h1>
-			<p>
-				You are about to leave the persona creation screenWARNING: If you do so, all changes will be
-				lost.
-			</p>
+	<InfoScreen title="Leaving Persona creation" onBack={() => (showWarningModal = false)}>
+		<div class="container info">
```

Comment:
I think this should probably be part a separate component. Pretty sure we use same styling elsewhere.

Reply:
I wasn't able to get to this today, but since the persona creation flow is not complete, I'll address it in the next phase.